app/models/amr_validated_reading.rb

```diff
@@ -67,13 +66,16 @@ def self.download_all_data
 
   def self.download_query_for_meter(meter)
     <<~QUERY
-      SELECT reading_date,
+      SELECT m.mpan_mprn,
+             CASE m.meter_type WHEN 0 THEN 'Electricity' WHEN 1 THEN 'Gas' WHEN 2 THEN 'Solar PV' WHEN 3 THEN 'Exported Solar PV' END,
```

Comment:
Should the meter type be translatable if it's a public facing report?
Also pondering if the enum mapping could be used rather than hardcoding here? Not sure if you can with it being pure sql though.

Reply:
I'm going to add a card for translating the downloads. will require some rework of the query and also how the CSV is generated at the moments its streaming the results from the database query directly to the response using a `COPY`